test/debugger/test-debugger-frame.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+require('../common');
+var repl = require('./helper-debugger-repl.js');
+
+repl.startDebugger('frames.js');
+
+var addTest = repl.addTest;
+
+// Continue
+addTest('c', [
+  /break in .*:3/,
+  /1/, /2/, /3/, /4/, /5/
+]);
+
+// List lines for frame 0
+addTest('list(1)', [
+  /2/, /3/, /4/
+]);
+
+// Select frame 1
+addTest('frame(1)', [
+  /Frame selected: 1/
+]);
+
+// List lines for frame 1
+addTest('list(1)', [
+  /4/, /5/, /6/
+]);
+
+// Invalid frame number
+addTest('frame(10)', [
+  /Invalid frame number/
+]);
+
+// Repl
+addTest('repl', [
+  /Ctrl/
+]);
+
+// Value of a is from selected frame
+addTest('a', [
+  /aaa/
+]);
```

Comment:
Can you add checks that moving back to frame 0 and inspecting `a` returns the right results?